src/content/en/updates/2016/03/play-returns-promise.md

```diff
@@ -71,5 +71,15 @@ of this new functionality. View it in a browser such as Chrome 50 that supports
 interface. Be forewarned: the page will automatically play music when you visit it. (Unless, of
 course, it doesn’t!)
 
+## Relevant bugs {: #bugs }
```

Comment:
I wouldn't call this a bug as it's actually part of the spec and there isn't much we can do. The bug will most likely be closed after this article is updated.